spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/ConfigMapPropertySource.java

```diff
@@ -42,14 +43,10 @@
 
 	private static final String PREFIX = "configmap";
 
-	public ConfigMapPropertySource(KubernetesClient client, String name) {
-		this(client, name, null, null);
-	}
-
 	public ConfigMapPropertySource(KubernetesClient client, String name, String namespace,
```

Comment:
I assumed deleting this constructor is safe because [only the tests invoke it](https://github.com/spring-cloud/spring-cloud-kubernetes/blob/master/spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsTest.java#L87). I will put it back.

Reply:
It's a public API. We have to assume someone is using it and would break with this change.

Reply:
I restored the constructor in the original public API.